tests/0045-subscribe_update.c

```diff
@@ -725,65 +784,84 @@ static void do_test_resubscribe_with_regex() {
          */
 
         TEST_SAY("Creating topic %s\n", topic1);
-        test_create_topic_wait_exists(NULL, topic1, 4, 1, 5000);
+        test_create_topic_wait_exists(NULL, topic1, 4, -1, 5000);
+        sleep_for(5);
 
         TEST_SAY("Creating topic %s\n", topic2);
-        test_create_topic_wait_exists(NULL, topic2, 4, 1, 5000);
+        test_create_topic_wait_exists(NULL, topic2, 4, -1, 5000);
+        sleep_for(5);
 
         TEST_SAY("Creating topic %s\n", topic_a);
-        test_create_topic_wait_exists(NULL, topic_a, 2, 1, 5000);
+        test_create_topic_wait_exists(NULL, topic_a, 2, -1, 5000);
+        sleep_for(5);
 
         test_conf_init(&conf, NULL, 60);
 
         rd_kafka_conf_set_events(conf, RD_KAFKA_EVENT_REBALANCE);
         rk    = test_create_consumer(group, NULL, conf, NULL);
         queue = rd_kafka_queue_get_consumer(rk);
 
+        sleep_for(3);
+
         /* Subscribe to topic1 */
         TEST_SAY("Subscribing to %s\n", topic1);
         test_consumer_subscribe(rk, topic1);
+
+        sleep_for(3);
```

Comment:
Question: In this test, why are we sleeping after subscribe and unsubscribe - there is no propagation step needed for subscribe/unsubscribe, so, where is the extra time requirement coming from? Are the await_assignment and await_revocations timing out? I see that test timeout multiplier is  not used in await_*, it's just 30